package.json

```
@@ -1,9 +1,13 @@
 {
-  "name": "iscroll",
+  "name": "@fulcrumapp/iscroll",
   "description": "Smooth scrolling for the web",
-  "version": "5.2.0",
-  "homepage": "http://iscrolljs.com",
-  "author": "Matteo Spinelli <matteo@cubiq.org> (http://cubiq.org)",
+  "version": "0.0.1",
+  "author": "",
+  "license": "ISC",
+  "repository": "https://github.com/fulcrumapp/common-node",
```

shouldn't this be `fulcrumapp/iscroll`?
```suggestion
  "repository": "https://github.com/fulcrumapp/iscroll",
```